php/Terminus/User.php

```diff
@@ -73,6 +75,12 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
+  public function instruments() {
```

Comment:
I'm ok if we want to lock all properties and associations behind an accessor method. But that still means creating this association in the constructor. Right now method this looks like a factory, not an accessor.